app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<!-- <h1>Tasks#show</h1>
+<p>Find me in app/views/tasks/show.html.erb</p> -->
+
+<% if @task %>
+<h3>
+  Id: <%=@task[:id] %>
+</h3>
+
+<h3>
+  Task: <%= @task[:name] %>
+</h3>
+
+<h4>
+  Description: <%= @task[:description] %>
+</h4>
+
+<h4>Completion Date: <%= @task[:completion_date] %></h4>
+
+<% else %>
+< h1>404: Not Found</h1>
```

Comment:
We'll talk about formally receiving a 404 HTTP Status request later on!

Reply:
Also, it's invalid html to have a space in that tag (you wrote `< h1>`, it should be `<h1>`